src/flownet/parameters/_relative_permeability.py

```diff
@@ -260,30 +271,47 @@ def render_output(self) -> Dict:
         partial_gen_wo = functools.partial(gen_wo, fast_pyscal=True)
         partial_gen_og = functools.partial(gen_og, fast_pyscal=True)
 
-        if self._swof and self._sgof:
-            with concurrent.futures.ProcessPoolExecutor() as executor:
-                for i, relperm in zip(  # type: ignore[assignment]
-                    parameters, executor.map(partial_gen_wog, parameters)
-                ):
-                    str_swofs += relperm.SWOF(header=False)
-                    str_sgofs += relperm.SGOF(header=False)
-        elif self._swof:
-            with concurrent.futures.ProcessPoolExecutor() as executor:
-                for i, relperm in zip(  # type: ignore[assignment]
-                    parameters, executor.map(partial_gen_wo, parameters)
-                ):
-                    str_swofs += relperm.SWOF(header=False)
-        elif self._sgof:
-            with concurrent.futures.ProcessPoolExecutor() as executor:
-                for i, relperm in zip(  # type: ignore[assignment]
-                    parameters, executor.map(partial_gen_og, parameters)
-                ):
-                    str_sgofs += relperm.SGOF(header=False)
+        if isinstance(self._interpolation_values, pd.DataFrame):
```

Comment:
As `interpolation_values` is will be evaluated as `True` when it is a `pd.DataFrame` and `False` when it is `None`, you can drop the `isinstance()` call.